Doc/about.rst

```diff
@@ -1,10 +1,10 @@
-=====================
-About these documents
-=====================
+========================
+About this documentation
+========================
 
 
-These documents are generated from `reStructuredText`_ sources by `Sphinx`_, a
-document processor specifically written for the Python documentation.
+These documents are generated from `reStructuredText`_ sources using `Sphinx`_,
+a documentation processor originally created for Python.
```

Comment:
```suggestion
Python's documentation is generated from `reStructuredText`_ sources
using `Sphinx`_, a documentation generator originally created for Python
and now maintained as an independent project.
```

Reply:
I like it, I'll commit it. A bit longer but clearer.